### PR TITLE
feat: Generate BFT pages with available information

### PR DESCRIPTION
**Pull request by sanjibansg**
This PR modifies the generation process for the function pages where even if all the information is not available, the pages will still be generated.

### CODE REVIEW

File: bft/dialects/types.py [hunk 1]
```diff
@@ -120,8 +120,6 @@ def mapping_for_case(self, case: Case) -> SqlMapping:
         dfunc_scalar = self.__scalar_functions_by_name.get(case.function, None)
         dfunc_aggregate = self.__aggregate_functions_by_name.get(case.function, None)
         dfunc = dfunc_scalar or dfunc_aggregate
-        if dfunc.unsupported:
```
Comment (richtia) [inline bft/dialects/types.py:123]: Can we keep this unsupported check somehow?  We still want to skip these tests from running with pytest.

Comment (richtia) [reply bft/dialects/types.py:123]: This is kind of a hacky way to get around this issue:
https://github.com/richtia/bft2/blob/c9d9e79a5430872ccafe35a8015412020edc553b/bft/dialects/types.py#L124-L131

Comment (richtia) [reply bft/dialects/types.py:123]: Actually a better way would be to do something like this
```
        if "PYTEST_CURRENT_TEST" not in os.environ:
            if dfunc is None:
                return None
        elif dfunc.unsupported:
            pytest.skip("Skipping unsupported function.")
```
make sure to import os

**Review by richtia — APPROVED** [review]

File: bft/dialects/types.py (hunk 1 above)
Comment (sanjibansg) [reply bft/dialects/types.py:123]: made the change, thanks!

**Pull request merged**